tests/js/spec/components/projects/bookmarkStar.spec.tsx

```diff
@@ -25,66 +18,65 @@ describe('BookmarkStar', function () {
   });
 
   it('renders', function () {
-    expect(wrapper).toSnapshot();
+    const {container} = render(
+      <BookmarkStar
+        organization={TestStubs.Organization()}
+        project={TestStubs.Project()}
+      />
+    );
+
+    expect(container).toSnapshot();
```

Comment:
I would just remove this - we know that it renders since the tests down the line pass

Reply:
This takes a screenshot of it though!

Reply:
Wait, this takes a screenshot, not a snapshot? These render tests are usually not very useful and just end up breaking every time we change a component with the fix often just being yarn test -u, they are imo very low value and we are already implicitly checking that our component renders in the tests below